nf_core/pipelines/lint/__init__.py

```diff
@@ -599,7 +599,11 @@ def run_linting(
     lint_obj._load_lint_config()
     lint_obj.load_pipeline_config()
 
-    if lint_obj.lint_config and not lint_obj.lint_config["nfcore_components"]:
+    if (
+        lint_obj.lint_config
+        and lint_obj.lint_config["nfcore_components"] is not None
+        and not lint_obj.lint_config["nfcore_components"]
```

Comment:
I think this is the correct condition. Could you check with a new created pipeline with `nf-core pipelines create`? Thanks!
```suggestion
        lint_obj.lint_config
        and (not lint_obj.lint_config["nfcore_components"]
        or lint_obj.lint_config["nfcore_components"] is not None)
```

Reply:
You're totally right

Reply:
when it's None we should check.
I think the best is to check if it is not None:
```
if (
        lint_obj.lint_config and 
        lint_obj.lint_config["nfcore_components"] is not None
)
```

Reply:
Actually, we should never see nf-core_components = True in `.nf-core.yml`, it is only possible to set it to false and skip this linting test. So if you prefer, we can check that it is False.

Reply:
Ah, I didn't knew...
In this case your code should be enough